packages/core/package.json

```diff
@@ -17,7 +17,7 @@
     "usedapp-generate-hooks": "./bin/usedapp-generate-hooks.js"
   },
   "dependencies": {
-    "@metamask/detect-provider": "^1.2.0",
+    "@metamask/detect-provider": "2.0.0",
```

Comment:
Is there a reason why the version is fixed?

Reply:
shouldn't be, I'll add the caret back! 👍